### PR TITLE
[capi-media-vision][ACR-1658] Update guides

### PR DESCRIPTION
**Pull request by tyoungroy**
Signed-off-by: Tae-Young Chung <ty83.chung@samsung.com>

Update guides according to changes acr-1658
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by omar-srbd — COMMENTED** [review]
Hello @tyoungroy,
I have a few corrections/suggestions for the files on this PR. I am listing them below.

**File:** _docs/application/native/guides/multimedia/image-classification.md_
1.	Line 8: “To enable your application to use the media vision inference functionality:”
Suggestion: “To enable your application to use the media vision inference functionality, follow these steps:”
2.	Line 40: “To classify an image:”
Suggestion: “To classify an image, follow these steps:”

**File:** _docs/application/native/guides/multimedia/pose-detection.md_
1.	Line 3: “…pose with Motion Capture (MoCap) file…”
Suggestion: “…pose with a Motion Capture (MoCap) file…”
2.	Line 13: “The Pose landmark detection models…”
Suggestion: “The pose landmark detection models…”
3.	Line 13: “…or public GitHub site such as…”
Suggestion: “…or on public GitHub site such as…”
4.	Line 62: “…in Media Vision…”
Suggestion: …in media vision…
5.	Line 89: “To enable your application to use the media vision inference functionality:”
Suggestion: “To enable your application to use the media vision inference functionality, follow these steps:”
6.	Line 120: “To detect human pose from an image:”
Suggestion: “To detect human pose from an image, follow the steps below:”

**Comment by tyoungroy** [comment]
Hi, @omar-srbd  i updated. please review it

**Review by omar-srbd — COMMENTED** [review]
@tyoungroy, please incorporate the suggestion at your earliest convenience.

File: docs/application/native/guides/multimedia/pose-detection.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In Tizen, human body pose landmarks and body parts are defined as follows:
...
+The pose landmark detection models are available in Open Model Zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or on public GitHub site such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
```
[the remark this answers is not in the document]
Comment (tyoungroy) [reply docs/application/native/guides/multimedia/pose-detection.md]: done

File: docs/application/native/guides/multimedia/image-classification.md [hunk 2]
```diff
@@ -111,12 +111,14 @@ To classify an image:
 4. Configure `g_engine_config` with classification model data to classify the image.
    The default engine is configured by the system. You can see the supported engines in Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f) applications).
 
-    In the following example, TensorFlow Lite model is used, and `data.tflite` and `label.txt` are in the `<OwnDataPath>`.
-	Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models):
+    In the following example, TensorFlow Lite model is used, and `data.tflite`, `meta.json`, and `label.txt` are in the `<OwnDataPath>`.
+	Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models).
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-classification.md:115]:
```suggestion
	Model data is available in open model zoo, such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models).
```

File: docs/application/native/guides/multimedia/image-classification.md [hunk 3]
```diff
@@ -111,12 +111,14 @@ To classify an image:
...
+  Its corresponding model meta file is available in [Meta file template](https://review.tizen.org/gerrit/gitweb?p=platform/core/api/mediavision.git;a=tree;f=meta-template;hb=refs/heads/tizen_6.5). :
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-classification.md:116]:
```suggestion
  Its corresponding model meta file is available in [Meta file template](https://review.tizen.org/gerrit/gitweb?p=platform/core/api/mediavision.git;a=tree;f=meta-template;hb=refs/heads/tizen_6.5):
```

File: docs/application/native/guides/multimedia/pose-detection.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ In Tizen, human body pose landmarks and body parts are defined as follows:
...
+The pose landmark detection models are available in Open Model Zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or on a public GitHub site such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/pose-detection.md:13]:
```suggestion
The pose landmark detection models are available in Open Model Zoo, such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or on a public GitHub site, such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
```

File: docs/application/native/guides/multimedia/pose-detection.md [hunk 5]
```diff
@@ -187,11 +187,12 @@ To detect human pose from an image:
    if (error_code != MEDIA_VISION_ERROR_NONE)
        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
    ```
-4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt`, which are described in the [Background](#background) section, are applied and the files are stored in `<OwnDataPath>`.
+4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt`, which are described in the [Background](#background) section, are applied and the files are stored in `<OwnDataPath>`. In addition, the model meta file `meta.json` which includes the model information is stored in `<OwnDataPath>`.
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/pose-detection.md:190]:
```suggestion
4. Configure `g_engine_config` with a body pose model data and its mapping file. Suppose that model data `data.tflite` and its mapping file `data_mapping.txt`, which are described in the [background](#background) section, are applied and the files are stored in `<OwnDataPath>`. In addition, the model meta file `meta.json` which includes the model information, is stored in `<OwnDataPath>`.
```

File: docs/application/native/guides/multimedia/pose-detection.md [hunk 6]
```diff
@@ -187,11 +187,12 @@ To detect human pose from an image:
...
 In the following example, all error check codes are omitted for the simplicity:
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/pose-detection.md:191]:
```suggestion
In the following example, all error check codes are omitted for simplicity:
```

File: docs/application/native/guides/multimedia/pose-detection.md [hunk 7]
```diff
@@ -226,29 +219,9 @@ In the following example, all error check codes are omitted for the simplicity:
...
-    MV_INFERENCE_INPUT_NODE_NAME,
-    inputNodeName);
-
-    error_code = mv_engine_config_set_array_string_attribute(imagedata.g_engine_config,
-    MV_INFERENCE_OUTPUT_NODE_NAMES,
-    outputNodeName, 1);
     ```
 
-    For more information on the configuration attributes such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications).
+    For more information on the configuration attributes such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications). For more information about model meta files, see [Meta file template](https://review.tizen.org/gerrit/gitweb?p=platform/core/api/mediavision.git;a=tree;f=meta-template;hb=refs/heads/tizen_6.5).
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/pose-detection.md:224]:
```suggestion
    For more information on the configuration attributes, such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications). For more information about model meta files, see [meta file template](https://review.tizen.org/gerrit/gitweb?p=platform/core/api/mediavision.git;a=tree;f=meta-template;hb=refs/heads/tizen_6.5).
```

**Review by omar-srbd — COMMENTED** [review]
@tyoungroy, please incorporate the changes at your earliest convenience.

**File:** _docs/application/native/guides/multimedia/pose-detection.md_

1.	Line 38: “…`-1` at third position denotes”
Suggestion: “…`-1` at **_the_** third position denotes” 
2.	Line 66: “The example starts with hips…”
Suggestion: “The example starts with **_the_** hips…”

**Comment by tyoungroy** [comment]
Regarding these commits, the update of this doc is because of deprecation of some apis. But your review comments are about everything of the docs. I have updated your last comments, but I don't think that more review comments are inappropriate.
Please check the last update.

**Comment by omar-srbd** [comment]
Hello @tyoungroy, apologies for the inconvenience and thank you so much for implementing the suggested changes.
I have gone through the changes you made and everything seems alright now.

**Review by omar-srbd — APPROVED** [review]
LGTM.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**